_articles/id/starting-a-project.md

```diff
@@ -2,7 +2,7 @@
 lang: id
 title: Memulai Proyek Open Source
 description: Pelajari lebih lanjut tentang dunia open source dan bersiap-siap untuk merilis proyek Anda sendiri.
-class: beginners
+class: pemula
```

Comment:
The class field should not be translated, see point 6 at https://github.com/github/opensource.guide/blob/master/docs/translations.md#starting-a-translation